cdap-ui/app/cdap/components/AbstractWidget/SchemaEditor/SchemaValidator/index.tsx

```diff
@@ -101,6 +101,11 @@ class SchemaValidatorProvider extends React.Component<ISchemaValidatorProviderPr
         const entireSchema = SchemaGenerator(schemaTree);
         cdapavsc.parse(entireSchema.schema, { wrapUnions: true });
       } catch (e) {
+        // TODO: CDAP-17152. We don't have specific fields that are duplicate today.
+        // We should improve the way we validate the avro schema
+        if (e.message.indexOf('duplicate field name') !== -1) {
```

Comment:
nit: we may also do `if (e.message.includes('duplicate field name')`

Reply:
We could but I don't see any specific advantage of using `includes` over `indexOf`. Do you see it as a better API?

Reply:
Since we're checking whether it includes plain string, I don't think it matters too much.